### ISSUE TITLE
Some sites are not banned :(

### ISSUE THREAD

**Issue by neradp** [e2guardian/e2guardian#185]
Hi...
Strange thing...  I have group with filtering on, but for example http://www.pornhub.com/ is not banned ...
url *porn*
body *fuck* *porn* *pussy* *gay* ....

Some subpages of this  is banned, but the main page is visible for all my network users (im school admin :))

I can add pornhub to banned site lists ... but what is  wrong with filtering engine ?

Im using version 3.4.0.3 and 4.0

**Comment by fredbcode** [comment]
Can you be more explicit please ?  

> but the main page is visible 
> I can add pornhub to banned site lists

If the main page is not banned ??

**Comment by neradp** [comment]
The question is : Why this page is not blocked by weightphrasemode or bannedurlregexplist ?

Im just suprised with this discovery.

**Comment by fredbcode** [comment]
Because this is not an url but a domain ?
About words I guess the "bads" are only on some/many pages not a the root who contains only codes 

E2Guardian is strict

**Comment by neradp** [comment]
Ok thank for explanation....

Is it posible to filter domain by regexp ?  not by domain lists .. ( if domain name is not tested by  urlregexp)

**Comment by fredbcode** [comment]
Yes use a syntax like this (toto.free.fr) in your banned expressions
But this is a strange request,maybe there is something wrong somewhere ...

Please ask your questions to the forum/mailing list

**Comment by fredbcode** [comment]
I mean in bannedregexpurllist

**Comment by onixx** [comment]
Hello,

I just wanted to say I have encountered this same issue and did some troubleshooting. I believe that the issue being encountered by neradp is related to configuration;
in e2guardian.conf, the default maxcontentfiltersize is set to 256. Some text/html request are larger than this. This was the case for the website referenced in the first post of this thread. 

It could be a consideration to increase the default as I noticed quite a few document are over this size these days. That is, even some google search results returned are larger and left unfiltered.
